### PR TITLE
Feat: video lib contexts

### PR DESCRIPTION
**Pull request by tewarimohit**
Features Added:-
- created frontend
- created pages.
- created context

### CODE REVIEW

File: src/Context/videoContext.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+const VideoProvider=({children})=>{
+
+const [videos,setVideos]=useState([]);
+
+
+useEffect(() => {
+    (async () => {
+        try {
+            const response = await axios.get("/api/videos");
+            console.log(response.data.videos);
```
Comment (deepak29-git) [inline src/Context/videoContext.js:15]: remove console

**Review by deepak29-git — APPROVED** [review]
your folder structure is good

File: src/Components/Header.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { Menu } from "@material-ui/icons";
+import { SearchRounded } from "@material-ui/icons";
+import { VideoCallRounded } from "@material-ui/icons";
+import { MicNoneOutlined } from "@material-ui/icons";
+import { AppsRounded } from "@material-ui/icons";
+import { NotificationsActiveRounded } from "@material-ui/icons";
```
Comment (arjunbharti) [inline src/Components/Header.js:6]: add them all in a single line.

File: src/index.js [hunk 3]
```diff
@@ -3,13 +3,25 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
+import { BrowserRouter } from "react-router-dom";
+import { WatchLaterProvider } from "./Context/WatchLaterContext";
+import { LikedVideosProvider } from "./Context/LikedVideosContext";
+import { VideoProvider } from "./Context/videoContext";
```
Comment (arjunbharti) [inline src/index.js:9]: Create an index.js file in context folder and import all these contexts in a single file. Then you can easily import all the contexts here easily

**Review by arjunbharti — APPROVED** [review]
Good file structuring. Just added a minor suggestion. Please go through it!

**Pull request merged**